packages/chart/src/components/BarChart.tsx

```diff
@@ -94,6 +97,44 @@ export default function BarChart({ xScale, yScale, seriesScale, xMax, yMax, getX
     return Math.ceil(context.measureText(text).width)
   }
 
+  function cleanData(data) {
+    let cleanedup = []
```

Comment:
nit pick: variable name could be improved

Reply:
I thought it was an obvious name... but changing it to cleanedupData